src/core/js/helpers.js

```diff
@@ -3,6 +3,16 @@ define([
     'core/js/adapt'
 ], function(Handlebars, Adapt){
 
+    var defaultAriaLevels = {
+        '_menu': 1,
+        '_menuItem': 2,
+        '_page': 1,
+        '_article': 2,
+        '_block': 3,
+        '_component': 4,
+        '_componentItem': 5
```

Comment:
as per https://github.com/adaptlearning/adapt_framework/pull/2496
```suggestion
        '_componentItem': 5,
        '_notify': 1
```